### DISCUSSION TITLE
Alpine Doc - Inner HTML section doesn't show the rendered result

### DISCUSSION THREAD

**Discussion by s-patompong** [alpinejs/alpine#1613] [7. Bugs (archived)]
I can't find the repo for Alpine doc so I need to create the issue here, hope that's ok.

Link: https://alpinejs.dev/essentials/templating#inner-html

**Problem**
The h1 tag doesn't get rendered in the result card (being blank). The screenshot is at [https://imgur.com/a/cEYbeNX](https://imgur.com/a/cEYbeNX)

I checked using the Chrome dev tool and can't see anything wrong, so not sure what's going on here.

**Comment by danddanddand** [comment]
It should work on latest build. Run Alpine.version in console and confirm you’re >= 3.1.0 (It wasn’t in initial 3.0.x release)

https://codepen.io/danddanddand/pen/WNpqPYm

**Reply by s-patompong** [reply]
The doc site is using version 3.0.5

```
<script defer src="https://unpkg.com/alpinejs@3.0.5/dist/cdn.min.js"></script>
```

@calebporzio need to update the Alpine version on the website to make this work properly then :)
